app/src/main/res/layout/activity_sample.xml

```diff
@@ -46,7 +46,17 @@
         android:layout_marginTop="16dp"
         app:keypadButtonSize="72dp"
         app:keypadShowDeleteButton="true"
-        app:keypadTextColor="@color/white"
-        app:keypadTextSize="18dp" />
+        app:keypadShowLetters="true"
+        app:keypadTextColor="#0000FF"
```

Comment:
The default color is white- I used a random color to test if other colors worked